client/src/components/AlertModal/AlertModal.js

```diff
@@ -0,0 +1,51 @@
+import {
+  Button as MuiButton,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+  styled,
+} from '@mui/material';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = styled(MuiButton)({
+  padding: '12px 24px',
+});
```

Comment:
no need for that,   just pass the padding in sx prop in the Button itself

Reply:
I think it is better to use one styled component than writing to more than one component same stylings

Reply:
i dont agree in this use case but not a deal breaker.